### DISCUSSION TITLE
Option for old explore reccomended category.

### DISCUSSION THREAD

**Discussion by TaromaruYuki** [community/community#27984] [Mobile]
I would waste 15-30 mins going through my explore reccomended tab on the explore tab. Seeing stuff that would be based on stars. I would see so many smaller projects and cool projects that I actually like. But with the current explore, there's popular repos, awesome lists, and a feed for starred repos, followed, and rarely repos that aren't super intresting to me.

What I'm suggesting is to have another category with the popular and awesome lists that did what the old app did: actually suggest repos that I would be interested in. I don't really use the app anymore because of the new U.I update. I would love to waste more time just looking at cool repos.

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Reply by TaromaruYuki** [reply]
That's all cool for desktop, but on mobile there isn't anything really like that. The stuff that are in my explore isn't really intresting and it shows status updates from repos i don't follow / star.

![Screenshot_20220802-181603_GitHub.png](https://user-images.githubusercontent.com/48397556/182483054-3a0aad71-b476-47d0-952a-75016330ddc5.png)

**Reply by jfullstackdev** [reply]
yup, it was just the start I think so, but they will further improve that I believe

**Comment by stevepopovich** [comment]
Hi @TaromaruYuki, thanks for the feedback! I will pass it on to the team.

**Comment by stevepopovich** [comment]
Just to follow up @TaromaruYuki:
1. The content you are seeing includes the original content that used to be in the feed, it just also includes other content now
2. We will be adding filters that will get you close to the original experience you desire.

I hope that helps!

**Reply by akamud** [reply]
On mobile I'm stuck with the new experience, which is too noisy and repetitive, for the reasons this original discussion mentioned. 
I would like to be able to filter the explore feed on mobile too.
